### PR TITLE
#38: Refactor the Manifests class to an immutable

### PR DESCRIPTION
**Pull request by proshin-roman**
I've made Manifests file immutable. As well I added `asMap()` method that returns a copy of an internal map of attributes. Method `keySet()` returns a copy of keys set too.

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](http://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#43](https://github.com/jcabi/jcabi-manifests/issues/43) is now in scope, role is `REV`

File: src/main/java/com/jcabi/manifests/MfMap.java [hunk 1]
```diff
@@ -40,15 +41,67 @@
  * @since 1.1
  * @see Manifests
  */
-public interface MfMap extends Map<String, String> {
+public interface MfMap {
```
Comment (yegor256) [inline src/main/java/com/jcabi/manifests/MfMap.java]: @proshin-roman Maybe it's not a good idea to change the public interface of classes that are already in production? Also, what do we gain from this?

**Comment by yegor256** [comment]
@proshin-roman see my comment. Also, what was the problem you were solving? What was wrong?

**Comment by proshin-roman** [comment]
@yegor256 that's strange that the ticket is not mentioned there automatically. This pull request has been created for the ticket #38 and it meets the requirements of the ticket.

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:jcabi/jcabi-manifests#43` is not assigned, can't get start time

**Comment by 0crat** [comment]
The job [#43](https://github.com/jcabi/jcabi-manifests/issues/43) is now out of scope

**Pull request merged**